soh/src/overlays/actors/ovl_Obj_Oshihiki/z_obj_oshihiki.c

```diff
@@ -567,9 +567,9 @@ void ObjOshihiki_Push(ObjOshihiki* this, PlayState* play) {
     f32 pushDistSigned;
     s32 stopFlag;
 
-    this->pushSpeed = this->pushSpeed + ((CVarGetInteger("gFasterBlockPush", 0) / 2) * 0.5) + 0.5f;
+    this->pushSpeed = this->pushSpeed + (CVarGetInteger("gFasterBlockPush", 0) * 0.25) + 0.5f;
     this->stateFlags |= PUSHBLOCK_PUSH;
-    this->pushSpeed = CLAMP_MAX(this->pushSpeed, 2.0f);
+    this->pushSpeed = CLAMP_MAX(this->pushSpeed, 2.0f + (CVarGetInteger("gFasterBlockPush", 0) * 0.5));
```

Comment:
the "not being pushed all the way" fix https://github.com/HarbourMasters/Shipwright/pull/1363/ changed this line from
```c
this->pushSpeed = CLAMP_MAX(this->pushSpeed, 3.0f);
```
to
```c
this->pushSpeed = CLAMP_MAX(this->pushSpeed, 2.0f);
```

with this logic it seems it's still possible to have that value be `3.0`

if i'm understanding correctly, this means the issue with not reaching the end was actually a problem with the earlier line, which was changed from
```c
this->pushSpeed = this->pushSpeed + (CVar_GetS32("gFasterBlockPush", 0) * 0.3) + 0.5f;
```
to
```c
this->pushSpeed = this->pushSpeed + ((CVar_GetS32("gFasterBlockPush", 0) / 2) * 0.5) + 0.5f;
```

Reply:
The fire temple block still worked with every block speed I tried. If I recall correctly the other pushable things work OK with the current enhancement, but I can check those today. Having the slider on +2 would put the top speed at 3.0, I feel like I tried that but I'll double check.

Reply:
All the blocks currently “work” but all of them are far slower than they should be for the highest setting IMO

Reply:
Can confirm with Block speed at +2 that one block still gets to the end of its track.

Reply:
so it must have been the `* 0.3` that was messing things up... (probably? i'm still not quite sure)

anyways, yes, we should update the speed of the rest of the things to use the same logic as this does (def want fast small blocks for the spirit temple sun block room)

Reply:
I've updated all the ones that are actually blocks. The other places are the shadow temple statue, the mirror statues in spirit temple, and the forest temple basement. I think the first one I had should have covered the spirit temple blocks, I'm fairly certain those are obj_oshihiki.

Reply:
Ugh nevermind, there's also a jya_block just for those blocks in child spirit temple...